### DISCUSSION TITLE
ARAB and Saudi channels Geo Blocked

### DISCUSSION THREAD

**Discussion by Diaa56** [iptv-org/community#302] [Q&A] [answered]
Hi,

all ARAB and Saudi channels says Geo Blocked and stream error on my Samsung Smart IPTV, please anyone can help to resolve this issue?

**Comment by freearhey** [comment, marked as answer]
https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#report-a-broken-stream
